bridges/TelegramBridge.php

```diff
@@ -0,0 +1,265 @@
+<?php
+class TelegramBridge extends BridgeAbstract {
+	const NAME = 'Telegram Bridge';
+	const URI = 'https://t.me';
+	const DESCRIPTION = 'Returns newest post from a public Telegram channel';
+	const MAINTAINER = 'VerifiedJoseph';
+	const PARAMETERS = array(array(
+			'username' => array(
+				'name' => 'Username',
+				'type' => 'text',
+				'exampleValue' => '@telegram',
+			)
+		)
+	);
+
+	const CACHE_TIMEOUT = 600; // 15 mins
```

Comment:
```suggestion
	const CACHE_TIMEOUT = 600; // 10 mins
```

Unless you mean

```suggestion
	const CACHE_TIMEOUT = 900; // 15 mins
```

Reply:
Thanks for the review, I did mean to the `CACHE_TIMEOUT` to 15 minutes.